File: CPU_2025/Heroi/heroi.py
import time

inicio = time.perf_counter()

nome = input().split(" ")
primeira_letra = nome[0][0]

aliterativo = True

for x in range(len(nome)):
	if not (nome[x].startswith(primeira_letra)):
		aliterativo = False

if(aliterativo):
	print("Viva!")
else:
	print("Bah!")

fim = time.perf_counter()

print(fim - inicio)

File: CPU_2025/Heroi/heroi.cpp
#include <iostream>
#include <string>
using namespace std;

int main () {

	string nome;
	char c;

	cin >> nome;
	c = nome[0];

	while(cin >> nome) {
		if(nome[0] != c) {
			cout << "Bah!" << endl;
			return 1;
		}
	}

	cout << "Viva!" << endl;
	return 0;
}
